### DISCUSSION TITLE
DBT project LOG

### DISCUSSION THREAD

**Discussion by dmitrii-khr** [AltimateAI/vscode-dbt-power-user#914] [Q&A]
Hi!
I can't see the DBT project log anymore in the output. 
What happened?
I have to use old versions of extension because of that.

**Comment by anandgupta42** [comment]
Hey @dmitrii-khr I am not sure what you exactly mean by "DBT project log". Can you give some examples and also the version of the extension where it stopped working?

**Comment by dmitrii-khr** [comment]
Hello! Thank you for your attention!
The last version where I have this useful feature is 0.28.4.
Version 0.29.0 already does not work for me.
I'm talking about this channel in the output window, where all SQL operations performed by DBT are listed. In my case, it is `jw_dwh dbt logs`, because `jw_dwh` is the name of dbt project.
![image](https://github.com/AltimateAI/vscode-dbt-power-user/assets/20114683/e098857d-85d5-4692-8f39-0372b6b7d6c8)
